lib/app.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (app) {
       req.checkContinue = true;
       fn(req, res);
     });
-    server.listen(port || process.env.PORT, cb);
+    server.listen(port || process.env.PORT || 0, cb);
```

Comment:
Nice fix. I'm assuming port 0 is a random port that acts the same as not supplying the port, but fixing a regression on Node 6. Is that accurate?

Reply:
Yeah.  I think they fixed a bug that a lot of people (us included) were relying on (since `undefined !== 0`), so they reverted the change in 8.